electroncash/web.py

```diff
@@ -98,9 +125,14 @@ def BE_tuple(config):
            )
 
 def BE_default_explorer():
-    return (DEFAULT_EXPLORER
-            if not networks.net.TESTNET
-            else DEFAULT_EXPLORER_TESTNET)
+    netname = networks.net.__name__
```

Comment:
Hmm. Would you mind terribly if I asked you to rewrite this to use:

```python
net = networks.net
if net is networks.TestNet:
     # stuff
elif net is networks.TestNet4:
    # otherstuff
# and so on...
```

`is` in Python tests if the object is the same exact object (in memory).... I feel that's more explicit .... even though probably equivalent to what you have here.  Just feels like it is "safer" and ore likely for an IDE to catch typos, etc.

Reply:
Yea, that seems much better, will update.